public/index.html

```diff
@@ -3,35 +3,174 @@
+                    </section>
+                    <input type="image" src="./banner/button__playlist__day.svg">
+                </section>
+            </section>
+            <section href="#" class="playlist__podcast">
+                <img src="./banner/playlist_back_draw.svg" alt="Music of interest" width="300" height="200"/>
+                <section class="flex__around__playlist">
+                    <section class="playlist__txt__block" >
+                        <span class="txt"><b>Подкасты</b></span>
+                        <span class="txt__14">О том, что вас интересует</span>
+                    </section>
+                    <input type="image" src="./banner/button__playlist__day.svg">
+                </section>
+            </section>
+        </section>
+        <section class="rec__txt flex__between">
+            <span class="main__txt">Albums for you</span>
+            <a href="#" class="tag txt">Show all</a>
+        </section>
+        <section class="recomendations">
+            <a href="#" class="playlist__podcast first__in__line">
+                <section class="playlist__for__you">
+                    <img src="./playlists/coldplay.png" alt="Coldpaly" width="225" height="225"/>
+                    <section class="playlist__txt__block__for__you" >
+                        <span class="txt"><b>Coldplay</b></span>
+                        <span class="txt__14">A Rush Of Blood To The Head</span>
+                        <span class="txt__14">2002</span>
+                    </section>
+                </section>
+            </a>
+            <a href="#" class="playlist__podcast">                
+                <section class="playlist__for__you">
+                    <img src="./playlists/tim.png" alt="Tim Story" width="225" height="225"/>
+                    <section class="playlist__txt__block__for__you" >
+                        <span class="txt"><b>Roedelius, Tim Story</b></span>
+                        <span class="txt__14">4 Hands</span>
+                        <span class="txt__14">2022</span>
+                    </section>
+                </section>                
+            </a>
+            <a href="#" class="playlist__podcast">                
+                <section class="playlist__for__you">
+                    <img src="./playlists/qouk.png" alt="Qouk" width="225" height="225"/>
+                    <section class="playlist__txt__block__for__you" >
+                        <span class="txt"><b>Quok</b></span>
+                        <span class="txt__14">Traveller</span>
+                        <span class="txt__14">2016</span>
+                    </section>
+                </section>                
+            </a>
+            <a href="#" class="playlist__podcast">                
+                <section class="playlist__for__you">
+                    <img src="./playlists//ila.png" alt="Ilyamazo" width="225" height="225"/>
```

Comment:
лишний слеш
Должно быть
```suggestion
                    <img src="./playlists/ila.png" alt="Ilyamazo" width="225" height="225"/>
```